packages/ui/src/components/Textarea/index.ts

```diff
@@ -0,0 +1 @@
+export * from './Textarea';
```

Comment:
To make codebase more consistent it would be great to export TextArea and its props separately